### PR TITLE
🌱 Update ADOPTERS.md for Andy Anderson

### PR DESCRIPTION
**Pull request by clubanderson**
## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by mikeshng** [comment]
Thanks @clubanderson Please sign off on your commit to pass the DCO check.

```
git commit -s --amend
git push -f origin patch-1
```

Could we add kubestellar as well?

**Comment by clubanderson** [comment]
signed - Thank you

**Comment by xuezhaojun** [comment]
Could we change to the same format of other adopters? For example: [Company/Project name](Company/Project official webside)

File: ADOPTERS.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ Below is a list of companies and projects that have adopted Open Cluster Managem
 - [RamenDR](https://github.com/RamenDR)
 - [Red Hat](https://www.redhat.com/)
 - [Xiao Hong Shu](https://www.xiaohongshu.com/)
+- [Andy Anderson](https://linkedin.com/in/clubanderson)
```
Comment (zhujian7) [inline ADOPTERS.md:13]: Could you move this line up so that it meets the alphabetical order.

Comment (clubanderson) [reply ADOPTERS.md]: done

Comment (clubanderson) [reply ADOPTERS.md]: please approve

**Comment by zhujian7** [comment]
Hi @clubanderson the second commit also needs to be signed off:
```
git commit -s --amend
git push -f origin patch-1
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -5,8 +5,10 @@
 Below is a list of companies and projects that have adopted Open Cluster Management(OCM). If you have been using OCM and your name is not on this list, send us a PR!
 
 - [Alibaba Cloud](https://www.alibabacloud.com/)
+- [Andy Anderson](https://linkedin.com/in/clubanderson)
```
Comment (zhujian7) [inline ADOPTERS.md:8]:
```suggestion
```

File: ADOPTERS.md [hunk 3]
```diff
@@ -5,8 +5,10 @@
...
 - [Ant Group](https://www.antgroup.com/)
 - [AppsCode Inc.](https://appscode.com)
```
Comment (zhujian7) [inline ADOPTERS.md:10]:
```suggestion
- [AppsCode Inc.](https://appscode.com)
- [Kubestellar](https://docs.kubestellar.io)
```

Comment (zhujian7) [reply ADOPTERS.md:10]: Sorry, I misunderstood, I think https://github.com/open-cluster-management-io/ocm/pull/501#issuecomment-2199542059 is right. It should be the Company/Project name and website.

**Comment by clubanderson** [comment]
closing, starting over.

**Pull request closed without merging**